### PR TITLE
Big Sur support

### PR DESCRIPTION
**Pull request by hbons**

### CODE REVIEW

**Comment by vancura** [comment]
@hbons We should merge this before the branches differ too much. Let's take a look at it this week.

File: FigmaSharp.Controls/FigmaSharp.Controls.Cocoa/Converters/Misc/EmbeddedWindowConverter.cs [hunk 1]
```diff
@@ -125,11 +125,12 @@ public override IView ConvertToView (FigmaNode currentNode, ViewNode parentNode,
 
 			BaseFakeWindowView fakeWindowView;
 
-			// Detect Big Sur (Darwin kernel version 20)
-			if (Environment.OSVersion.Version.Major >= 20)
+#if OS_MAC_BIGSUR
+			if (MacSystemInformation.OsVersion < MacSystemInformation.BigSur)
```
Comment (hbons) [inline FigmaSharp.Controls/FigmaSharp.Controls.Cocoa/Converters/Misc/EmbeddedWindowConverter.cs]: I think this should be flipped? :)

Comment (netonjm) [reply FigmaSharp.Controls/FigmaSharp.Controls.Cocoa/Converters/Misc/EmbeddedWindowConverter.cs]: It was flipped in the next commit 😄

**Review by netonjm — CHANGES_REQUESTED** [review]
The PR needs address some changes. I added some small fixes to don't break build in systems < BigSur, now we have a new configuration DebugBigSur and ReleaseBigSur to enable this new API's

**Comment by netonjm** [comment]
@hbons any update on this PR?

**Comment by hbons** [comment]
nope... no time yet.

**Pull request closed without merging**